### DISCUSSION TITLE
Disable all map mouse events

### DISCUSSION THREAD

**Discussion by ynkfish** [liodali/osm_flutter#567] [General]
Hi,

I'm developing a flutter app where initially the OSM map is used as backdrop behind my main menu.
However it seems like the map component is taking all mouse events and my menu buttons are not hoverable/clickable.
I'm debugging with chrome, I haven't tried android/iOS yet.

I have tried putting the map in an AbsortPointer and IgnorePointer, but it doesn't work.

Is there a way to disable all mouse interaction with the map?

As an alternative I also tried to implement my menu interaction with mixin listener onSingleTap(GeoPoint position), but I don't see a way to translate a GeoPoint position to a screen coordinate to check if I need to trigger one of my buttons...

Any ideas would be greatly appreciated

**Comment by liodali** [comment]
that issue due to platformview in web
use pointer_interceptor package 
https://pub.dev/packages/pointer_interceptor
around pointerInterceptor widget will all component that you want to be clickable

**Reply by ynkfish** [reply]
Thanks, that works like a charm!
